.github/workflows/redeploy/redeploy_specific.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+
+iplist=$1
+
+for ip_address in $iplist
+do
+    echo $ip_address
+    ssh-keyscan -H $ip_address >> ~/.ssh/known_hosts
+    sleep 2
+
+    if [ "$ip_address" == "167.172.70.98" ] || [ "$ip_address" == "cloudretro.io" ] 
+    then
+        launchcommand="coordinator > /tmp/startup.log"
+    else
+        launchcommand="Xvfb :99 & worker --coordinatorhost cloudretro.io --zone \$zone > /tmp/startup.log"
+    fi
+
+    ssh root@$ip_address "mkdir -p /cloud-game/configs"
+    rsync ./.github/workflows/redeploy/config.yaml root@$ip_address:/cloud-game/configs/config.yaml
+    run_content="'#! /bin/bash
+    echo $PASSWORD | docker login https://docker.pkg.github.com --username $USERNAME --password-stdin; 
+    ufw disable;
+    docker system prune -f;
+    source /etc/profile; 
+    docker pull docker.pkg.github.com/giongto35/cloud-game/cloud-game:latest;
+    docker rm cloud-game -f;
+    docker run --privileged -d --network=host --env DISPLAY=:99 --env MESA_GL_VERSION_OVERRIDE=3.3 -v cores:/usr/local/share/cloud-game/assets/cores -v /cloud-game/configs:/usr/local/share/cloud-game/configs -v /cloud-game/games:/usr/local/share/cloud-game/assets/games -v /cloud-game/cache:/usr/local/share/cloud-game/assets/cache --name cloud-game docker.pkg.github.com/giongto35/cloud-game/cloud-game bash -c \"$launchcommand\"'"
```

Comment:
Might split long lines this: https://github.com/giongto35/cloud-game/blob/master/scripts/install.sh#L11-L16

Reply:
ah that's right, I can update like that

Reply:
Ok, will change if you won't.